spec/index.bs

```diff
@@ -3521,6 +3542,8 @@ A {{GPUProgrammableStageDescriptor}} describes the entry point in the user-provi
                         :: the |binding| is a non-comparison sampler
                         : {{GPUSamplerBindingType/"comparison"}}
                         :: the |binding| is a comparison sampler
+                        : {{GPUSamplerBindingType/"minmax"}}
+                        :: the |binding| is a min/max sampler
```

Comment:
does wgsl really need to differentiate minmax samplers? can't we do without, like we have for filtering/nonfiltering, by reflecting which textures and samplers are used together?

Reply:
the filtering-non-filtering is in the same boat: we want to validate earlier than draw time, and some combinations of sampler-texture are not valid

Reply:
WGSL doesn't differentiate filtering and non-filtering, it only differentiates comparison and non-comparison. This text ("the binding is a min/max sampler") implies that WGSL would have to start differentiating minmax samplers as well.

Reply:
You are right, I should not have touched this section at all. Removed it now, and added an internal slot for minmax samplers.